app/configurations/config.hb.js

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable */
+import configMerger from '../util/configMerger';
+
+const CONFIG = 'hb';
+const APP_TITLE = 'Mobil in Herrenberg';
+const APP_DESCRIPTION = '';
+
+const API_URL = process.env.API_URL || 'https://api.mobil-in-herrenberg.de';
+const MAP_URL = process.env.MAP_URL || 'https://maps.wikimedia.org/osm-intl/';
+const GEOCODING_BASE_URL = `https://pelias.locationiq.org/v1`;
+const LOCATIONIQ_API_KEY = process.env.LOCATIONIQ_API_KEY;
+
+const walttiConfig = require('./waltti').default;
```

Comment:
if the config is for Herrenberg, do we really need the waltti config imported here? I see that later in the config merger this is used as some default skeleton and then some specifics are overwritten. Maybe use a different name then for the variable, like defaultConfig, or originalConfig

Reply:
We can use the herrenberg config without the waltti one, but we must copy the transport modes.